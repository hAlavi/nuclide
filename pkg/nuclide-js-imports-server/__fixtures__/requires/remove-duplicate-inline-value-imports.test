import {a, a, b as c, b} from 'a';

const x = a(b, c);
