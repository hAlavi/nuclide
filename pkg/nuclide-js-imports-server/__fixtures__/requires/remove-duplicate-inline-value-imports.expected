import {a, b, b as c} from 'a';

const x = a(b, c);
